cost/azure/idle_compute_instances/azure_idle_compute_instances.pt

```diff
@@ -239,6 +431,10 @@ policy "azure_resource_policy" do
       field "vmname" do
         label "Name"
       end
+      field "savings" do
+        label "Estimated Monthly Savings"
+        format "currency"
```

Comment:
There is no format for currency.  it should be removed to not confuse the reader.